src/spikeinterface/extractors/neoextractors/blackrock.py

```diff
@@ -92,16 +94,18 @@ class BlackrockSortingExtractor(NeoBaseSortingExtractor):
     def __init__(
         self,
         file_path,
-        sampling_frequency: Optional[float] = None,
         stream_id: Optional[str] = None,
         stream_name: Optional[str] = None,
+        sampling_frequency: Optional[float] = None,
+        nsx_to_load: Optional[int | list | str] = None,
```

Comment:
the typing of this seems strange, can you double check @luiztauffer?

Sam is going to vacations so if he does not have time to check it before that I will merge this next Monday but just double check this.

Reply:
@h-mayorquin this comes from here: https://github.com/NeuralEnsemble/python-neo/blob/c425a529958d582919607a0eb940154a4b5c4d7e/neo/rawio/blackrockrawio.py#L97